cmake/modules/OpmPythonAddEmbeddedTests.cmake

```diff
@@ -1,16 +1,198 @@
-function (add_opm_embedded_tests)
+
+function (add_opm_embedded_tests_from_opm_common)
   if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     set(_libs mocksim opmcommon ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
 
-    foreach( test test_msim test_msim_ACTIONX test_msim_EXIT)
-      opm_add_test(${test} SOURCES embedded/tests/msim/${test}.cpp
+    foreach( test msim msim_ACTIONX msim_EXIT)
```

Comment:
test can be removed, only `test_msim_ACTIONX.cpp  test_msim.cpp  test_msim_EXIT.cpp` are in the embedded/tests/msim folder

Reply:
`test` is here the loop variable, right? It is not a specific test case

Reply:
ah yes, you're right :+1:

Reply:
Thanks for the review :smile: I am merging this now.